packages/framework/src/Model/Stock/StockSettingsDataFacade.php

```diff
@@ -25,13 +26,14 @@ public function __construct(
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Stock\StockSettingsData $stockSettingsData
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
      */
-    public function edit(StockSettingsData $stockSettingsData): void
+    public function edit(StockSettingsData $stockSettingsData, DomainConfig $domainConfig): void
```

Comment:
`AdminDomainTabsFacade` is still left in the contructor

Reply:
removed 👍